docs/source/run/parameters.rst

```diff
@@ -60,6 +60,11 @@ General parameters
     This option is only available in serial runs, in parallel runs, please use more GPU to achieve
     the same effect.
 
+* ``hipace.openpmd_backend`` (`string`) optional
```

Comment:
I know there's a bit more details later, but I think it's good to have this here
```suggestion
* ``hipace.openpmd_backend`` (`string`) optional (default `h5`)
```

Reply:
Ok, that's fine with me, too. Thanks!